RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+Release 1.3
```

Comment:
You'll want to merge in the latest release notes from `develop` (which is currently empty). Also, I am (pretty) sure you don't have to create labels like this because the Platform gang clears this file with every release — just adding a single bullet item is sufficient.

Reply:
Ahha - I thought it was a running list. 👍